pkg/openstack/rabbitmq.go

```diff
@@ -136,6 +136,21 @@ func reconcileRabbitMQ(
 									Name:  "HOME",
 									Value: "/var/lib/rabbitmq",
 								},
+								{
+									// The various /usr/sbin/rabbitmq* scripts are really all the same
+									// wrapper shell-script that performs some "sanity checks" and then
+									// invokes the corresponding "real" program in
+									// /usr/lib/rabbitmq/bin.  The main "sanity check" is to ensure that
+									// the user running the command is either root or rabbitmq.  Inside
+									// of an openshift pod, however, the user is neither of these, so
+									// the wrapper script will always fail.
+
+									// By putting the real programs ahead of the wrapper in PATH we can
+									// avoid the unnecessary check and just run things directly as
+									// whatever user the pod has graciously generated for us.
+									Name:  "PATH",
+									Value: "/usr/lib/rabbitmq/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+								},
```

Comment:
the alternitive would be to use tcib_envs: like this
https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/base.yaml#L43C1-L45
to set this in 
https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/rabbitmq/rabbitmq.yaml